apps/rush/src/utilities/RushHooksManager.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  RushHooks,
+  Utilities,
+  RushHookName
+} from '@microsoft/rush-lib';
+
+export default class RushHooksManager {
+  public constructor(private _rushHooks: RushHooks) {
+  }
+
+  public handle(hookName: RushHookName): void {
+    this._rushHooks.get(hookName).forEach((script) => {
+      Utilities.executeCommandOnShell(script,
```

Comment:
It is up to individual script to decide whether it wants to print out something.

---
In reply to: [117121270](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117121270) [](ancestors = 117121270)